typescript/sdk/src/metadata/chainMetadataTypes.ts

```diff
@@ -194,6 +194,15 @@ export const ChainMetadataSchemaObject = z.object({
     .boolean()
     .optional()
     .describe('Whether the chain is considered a testnet or a mainnet.'),
+  index: z
+    .object({
+      from: z
+        .number()
+        .optional()
+        .describe('The block to start any indexing from.'),
+    })
+    .optional()
+    .describe('Indexing settings for the chain.'),
```

Comment:
My recommendation would have been for this to be under `blocks.indexFrom` but this is okay too